metacat-functional-tests/metacat-test-cluster/docker-compose.yml

```diff
@@ -116,10 +98,8 @@ services:
         depends_on:
             - hive-metastore-db
             - postgresql
-            - cassandra
-            - druid
         environment:
-            - TARGETS=postgresql:5432,cassandra:9042,hive-metastore-db:3306,druid:8081
+            - TARGETS=postgresql:5432,hive-metastore-db:3306
```

Comment:
Add comments that we will reinstate the cassandra and druid tests once we figure a lightweight docker image for the same.

Reply:
will do.